### DISCUSSION TITLE
Feat: Medusa Sendgrid simple templates for quick start

### DISCUSSION THREAD

**Discussion by luluhoc** [medusajs/medusa#5928] [Feature Requests]
Hey,

Please include SendGrid templates for all transactional emails in the Medusa repo so people get a quick start.

**Comment by kriss145** [comment]
@luluhoc I'm working with the plugin where we will have ready all templates out of the box and also more another features which are on my roadmap. If u want some features, add your ideas in the discussions page :)

https://github.com/codee-sh/medusa-plugin-notifications
